quickstarts/oma-data-gov/config.yml

```diff
@@ -0,0 +1,37 @@
+# Name of the quickstart (required)
+name: oma-data-governance
+
+# Displayed in the UI (required)
+title: OMA Data Ingest Governance
+
+# Long-form description of the quickstart (required)
+description: |
+  Data ingest governance is a practice of ensuring optimal value for telemetry data collected by an organization particularly a complex organization with numerous business units and working groups.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  *Baseline*, *Optimize*, *Forecast* Your Telemetry Ingest
```

Comment:
This summary should be written the like the description, more use friendly product language as it's used in the preview tiles in both the external and internal I/O catalogs.